deploy/olm-catalog/service-telemetry-operator/metadata/annotations.yaml

```diff
@@ -2,6 +2,6 @@ annotations:
   operators.operatorframework.io.bundle.channel.default.v1: stable
   operators.operatorframework.io.bundle.channels.v1: stable
   operators.operatorframework.io.bundle.manifests.v1: manifests/
-  operators.operatorframework.io.bundle.mediatype.v1: plain
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
```

Comment:
Had to look this up, but seems right.

Reply:
Tea that old annotations.yaml was from an earlier build, and was never in use.